### ISSUE TITLE
ok ru stream/download link still will work after 2/3 days or after one month?

### ISSUE THREAD

**Issue by tvserial** [KhunHtetzNaing/xGetter#59]
ok ru stream/download link still will work after 2/3 days or after one month? i mean direct link erpire or not after few days?/

**Comment by osmanghaziurdu** [comment]
Yes Direct Link Expires. In Direct Link You Can Clearly See That "expires=" property is present.
